### DISCUSSION TITLE
How to show associated attachment model images in forms?

### DISCUSSION THREAD

**Discussion by shebaoting** [filamentphp/filament#9474] [Help]
Currently there is the IdentityVerification model. IdentityVerification and Attachment are one-to-many relationships. I want to display the Attachment of the current message in the resource edit page of IdentityVerification, what should I do?

```
    public function attachments()
    {
        return $this->hasMany(Attachment::class, 'model_id');
    }
```

```
 Section::make('info')
                    ->description('info')
                    ->aside()
                    ->schema([
                        Repeater::make('attachments')
                            ->relationship()
                            ->schema([
                                FileUpload::make('paths')->image()
                            ])
                            ->addable(false)
                            ->deletable(false)
                    ])
```

`paths` is the field associated with the model attachment. It is the address of an image. I want it to show up. I don't know how to do it.

**Comment by VincvanAs** [comment]
Try ->relationship('attachments')

**Comment by leandrocfe** [comment]
This is supposed to work. Could you explain more about the issue?
